Lusas_oM/Versioning_63.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Namespace": {
+    "ToNew": {
+    },
+    "ToOld": {
+    }
+  },
+  "Type": {
+    "ToNew": {
+
+    },
+    "ToOld": {
+    }
```

Comment:
See https://github.com/BHoM/Versioning_Toolkit/blob/23b5ea911317c50e37a0210533905f11da0af35a/BHoMUpgrader62/Upgrades.json#L14C14-L14C14 for an example

Reply:
Does not the change made result in a structural change. Meaning this is not enough?

Reply:
After discussion, conversion made in `Versioning_Toolkit` instead.